Remotion/ObjectBinding/Web/UI/Controls/BocList.cs

```diff
@@ -1035,7 +1040,7 @@ protected override void OnPreRender (EventArgs e)
         if (currentRow == null)
           _newPageIndex = null;
         else
-          _newPageIndex = currentRow.SortedIndex / _pageSize.Value;
```

Comment:
can we link _pageSize != null with IsPagingEnabled = true?

Reply:
It already is, but this doesn't help in this case